spec.html

```diff
@@ -37681,9 +37686,9 @@ <h1>The Atomics Object</h1>
     <emu-clause id="sec-abstract-operations-for-atomics">
       <h1>Abstract Operations for Atomics</h1>
 
-      <emu-clause id="sec-validatesharedintegertypedarray" aoid="ValidateSharedIntegerTypedArray">
-        <h1>ValidateSharedIntegerTypedArray ( _typedArray_ [ , _waitable_ ] )</h1>
-        <p>The abstract operation ValidateSharedIntegerTypedArray takes argument _typedArray_ and optional argument _waitable_ (a Boolean). It performs the following steps when called:</p>
+      <emu-clause id="sec-validateintegertypedarray" aoid="ValidateIntegerTypedArray" oldid="sec-validatesharedintegertypedarray">
+        <h1>ValidateIntegerTypedArray ( _typedArray_ [ , _waitable_ ] )</h1>
+        <p>The abstract operation ValidateIntegerTypedArray takes argument _typedArray_ and optional argument _waitable_ (a Boolean). It performs the following steps when called:</p>
```

Comment:
it seems weird for me (both before and after this change) that an abstract operation called "validate" returns something. is there a better name for this abstract op that conveys that?

Reply:
I'd consider that outside the scope of this PR.

One option is to tack on another verb, like `ValidateIntegerTypedArrayAndGetBuffer` and `ValidateAtomicAccessAndGetIndex`, that's pretty verbose.

Another option is to `ToValidIntegerTypedArrayBuffer`, but it might be hard to understand the intent of the AO without reading what it does.

In any case, we should do those as a follow-on PR.

Reply:
Maybe `GetSharedIntegerTypedArrayBuffer`?

I'm not sure it's outside the scope; you're renaming the op in this PR, so any concerns about the name seem thus in scope of the PR.

Reply:
It no longer applies to shared buffers only, which is why this renames that op to delete the word "shared". It doesn't follow to me that a complete rename is also in scope.

Also `GetIntegerTypedArrayBuffer` doesn't capture most of what the AO does: validating the kind of TypedArray the argument is depending on the parameters.